internal/cmd/operator-sdk/alpha/config3alphato3/convert_config_3-alpha_to_3.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2021 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config3alphato3
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"io/ioutil"
+	"path"
+	"regexp"
+	"strings"
+	"text/template"
+
+	"golang.org/x/mod/modfile"
+	"sigs.k8s.io/kubebuilder/v3/pkg/model/resource"
+	"sigs.k8s.io/yaml"
+)
+
+var (
+	v3alpha   = "3-alpha"
+	versionRe = regexp.MustCompile(`version:[ ]*(?:")?3-alpha(?:")?`)
```

Comment:
Multiple spaces and double quotes were never created by the CLI, but I guess they don't hurt and will success if they user added them manually.

Reply:
That's what I figured, i.e. if they migrated `"2" -> 3-alpha` by hand.